src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -96,6 +96,9 @@ void sl_ble_init()
     //  initializing the application events map
     rsi_ble_app_init_events();
     memcpy(randomAddrBLE, &randomAddr, 6);
+    ChipLogError(DeviceLayer, "%s: BLE : randomAddr %" PRIu64 "\n", __func__, randomAddr);
```

Comment:
Those aren't errors. Used `ChipLogDetails` or `ChipLogProgress`

You can drop the` __func__ ` it isn't too useful here.

Reply:
Removed the logs